.github/workflows/testBuildHybrid.yml

```diff
@@ -47,6 +47,10 @@ on:
         value: ${{ jobs.androidHybrid.outputs.S3_APK_PATH }}
       IOS_LINK:
         value: ${{ jobs.iosHybrid.outputs.IOS_PATH }}
+      ANDROID:
+        value: ${{ jobs.androidHybrid.result }}
+      IOS:
+        value: ${{ jobs.iosHybrid.result }}
```

Comment:
Could you also add the result into the output of those actions and avoid changing the actionlint?